update-annotations-table.R

```diff
@@ -20,17 +20,57 @@ library("here")
 #' @return The module or key name
 get_info <- function(id, info = c("module", "key")) {
   info <- match.arg(info)
-  pattern <- "^[^-]+-([[:alnum:]]+)\\.([[:alnum:]]+)-[0-9\\.]+"
+  if (grepl("-[0-9\\.]+", id)) {
+    pattern <- "^[^-]+-([[:alnum:]]+)\\.([[:alnum:]]+)-[0-9\\.]+"
+  } else {
+    pattern <- "^[^-]+-([[:alnum:]]+)\\.([[:alnum:]]+)"
+  }
```

Comment:
I think you could make the version part optional in the regex and avoid having to branch here, but this works too.

Reply:
With the requiring of versions (see reprogrammedCellType discussion), I can remove the extra branch logic now. You're totally right that the branching would be unnecessary with an update to the regex pattern, though.